### PR TITLE
Some small changes for clarity

### PR DESCRIPTION
**Pull request by trodrigues**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,11 +44,11 @@ Also, please avoid bundling `node_modules` or other installed dependency directo
 
 We will grade your project after submission and eventually invite you for a remote live interview with our engineers.
 
-Depending on the role you are applying for and your level of expertise, you have the flexibility to focus on implementing either the backend or the client application.
-We welcome submissions with basic functionality on the side of the stack you are less familiar with, allowing us to assess your skills in your preferred domain.
-If you opt for this approach, please specify in your notes which application you would like us to thoroughly review.
-Alternatively, if you feel confident in both the backend and client application, we encourage you to state clearly in your notes that you would like a comprehensive review of both.
-We are excited to thoroughly evaluate your work across the entire stack.
+Depending on the role you are applying for and your level of expertise, you have the flexibility to put a stronger focus in the implementation of either the backend or the client application. We welcome submissions with basic functionality on the side of the stack you are less familiar with, allowing us to assess your skills in your preferred domain.
```
Comment (akos-hrvth) [inline README.md:47]: I love the changes. If we still require a bare bones non-focus stack, I would rephrase even further to make it super clear.

> Depending on the role you are applying for and your level of expertise, you have the flexibility to put a stronger focus in the implementation of either the backend or the client application. **We require submissions to have basic functionality on the side of the stack you are less familiar with, however you are welcome to focus your efforts on your focus area, allowing us to assess your skills in your preferred domain.**

Comment (trodrigues) [reply README.md:47]: Nice, I like that! I'll update it. Thanks!
